src/adapters/auth-adapters.ts

```diff
@@ -83,10 +83,12 @@ This function login a user with the provided information.
 */
 export const loginAdapter = async (loginInput: LoginInputs ) => {
     const url =`http://localhost:8000/api/v1/user/authentication`
-    const data = await fetchAPIData(url , 'POST', loginInput)
-    setCookie('shelf-share-session', JSON.stringify(data), 1)
+    const data = await fetchAPIData(url , 'POST', loginInput, false)
+    if (data) {
+        setCookie(cookieName, JSON.stringify(data.token), 1)
```

Comment:
I like the error handling you added in here. We do want all of data to be in the cookie though, so can you change that part back?

Reply:
Can you add a check for data for the signUpAdapter as well?

Reply:
done